python/lsst/obs/hsc/transformRegistry.py

```diff
@@ -131,9 +136,11 @@ class HscDistortionConfig(Config):
                                    1.98549941035e-37,
                                    -8.74305862185e-38,
                                    ])
-    skyToCcdOrder = Field(dtype=int, doc="Polynomial order for conversion from sky to CCD (x and y identical)",
+    skyToCcdOrder = Field(dtype=int,
+                          doc="(ignored) Polynomial order for conversion from field angle to focal plane",
```

Comment:
If these are ignored, can we just remove them?

Reply:
OK. As long as you don't mind. In the long run I'd much rather have usable coefficients instead of using an iterative inverse, but that's out of scope for this ticket.